test/jdk/java/util/logging/FileHandlerAccessTest.java

```diff
@@ -41,10 +39,10 @@
  */
 
 public class FileHandlerAccessTest {
-    public static void main(String[] args){
+    public static void main(String[] args) {
         if (!(args.length == 2 || args.length == 1)) {
             System.out.println("Usage error: expects java FileHandlerAccessTest [process/thread] <count>");
-            System.exit(1);
+            return;
```

Comment:
Ah - sorry - since this is a test, instead of return you should probably throw an exception - e.g.:
```
throw new IllegalArgumentException("Usage error: expects java FileHandlerAccessTest [process/thread] <count>");
```

Reply:
Done! Thanks Daniel